### PR TITLE
Bump to lts-14.27

### PR DESCRIPTION
**Pull request by arianvp**
Depends on https://github.com/wireapp/hscim/pull/44

I was wondering if bumping would work without too many changes; and it seems it did :)

### CODE REVIEW

**Comment by arianvp** [comment]
We seem to hardcode LTS in CI; do we have to? it's a bit annoying we have to do this in two places

https://github.com/wireapp/wire-server-private/blob/d4594cf03c1756381639d86e513ab883c8587450/ci/dockerfiles/ubuntu14-ghc-8/Dockerfile#L7

File: stack.yaml [hunk 1]
```diff
@@ -50,8 +50,8 @@ nix:
 allow-newer: true
 
 extra-deps:
-- servant-swagger-1.1.7.1@rev:2
-- swagger2-2.5
+# - servant-swagger-1.1.7.1@rev:2
+# - swagger2-2.5
```
Comment (mheinzel) [inline stack.yaml]: what happened with swagger?

Comment (arianvp) [reply stack.yaml]: revision two was added to `14.27` https://www.stackage.org/lts-14.27/package/servant-swagger-1.1.7.1

`swagger2-2.5` I accidentally dropped. Good catch. However it still compiles. What was released in `2.5` that we need? 

I'll just add it back for now

Comment (arianvp) [reply stack.yaml]: @fisx was there a specific reason why we were on 2.5 ?   Dropping it to 2.4 worked fine (no dependency conflicts or compiler errors)

Comment (arianvp) [reply stack.yaml]: The `swagger2-2.5` release on hackage does not even compile for me.  I'm confused.  Given @fisx is maintainer of this package maybe he knows what is up here? https://github.com/wireapp/wire-server/pull/1030/commits/86ed4053cce9038eeb8fdca450ca889ace49f209

**Comment by fisx** [comment]
> We seem to hardcode LTS in CI; do we have to? it's a bit annoying we have to do this in two places
> 
> https://github.com/wireapp/wire-server-private/blob/d4594cf03c1756381639d86e513ab883c8587450/ci/dockerfiles/ubuntu14-ghc-8/Dockerfile#L7

Yes, that looks redundant, but I don't think it's necessary that the resolver there and the one of wire-server are the same.  In the docker image it is just used to build some haskell-related executables, no?

File: stack.yaml (hunk 1 above)
Comment (fisx) [reply stack.yaml]: [swagger2-2.5 is nice, but not important](https://github.com/GetShopTV/swagger2/releases/tag/v2.5).  I'll see if i can fix the issue upstream, then we can just wait for the next release.

**Pull request merged**